vignettes/sf1.Rmd

```diff
@@ -159,7 +159,7 @@ We will now discuss each of these three classes.
 
 ## sf: objects with simple features
 
-As we usually do not work with geometries of single simple features,
+As we usually do not work with geometries of single _simple features_,
```

Comment:
why italic here, but code font in 44?

Reply:
Correct. I will replace it with `simple features`. Thanks.